internal/cmd/inventoryconsumer/cmd.go

```diff
@@ -69,44 +69,38 @@ func handler(ctx context.Context, msg kafka.Message) {
 	isConnected := event.Host.SystemProfile.RHCID != ""
 
 	if isConnected {
-		switch eventType {
-		case "created":
-			logger.Info().Msg("setting up new host for remote host configuration")
-			reqID, _ := util.Kafka.GetHeader(msg, "request_id")
-			logger = logger.With().Str("request_id", reqID).Str("host_id", event.Host.ID).Str("org_id", event.Host.OrgID).Logger()
-			fallthrough
-		case "updated":
-			var defaultState map[string]string
-			if err := json.Unmarshal([]byte(config.DefaultConfig.ServiceConfig), &defaultState); err != nil {
-				logger.Error().Err(err).Msg("cannot unmarshal service config")
-				return
-			}
+		reqID, _ := util.Kafka.GetHeader(msg, "request_id")
+		logger = logger.With().Str("request_id", reqID).Str("host_id", event.Host.ID).Str("org_id", event.Host.OrgID).Logger()
+		var defaultState map[string]string
+		if err := json.Unmarshal([]byte(config.DefaultConfig.ServiceConfig), &defaultState); err != nil {
+			logger.Error().Err(err).Msg("cannot unmarshal service config")
+			return
+		}
 
-			profile, err := db.GetOrInsertCurrentProfile(event.Host.OrgID, db.NewProfile(event.Host.OrgID, event.Host.Account, defaultState))
-			if err != nil {
-				logger.Error().Err(err).Str("org_id", event.Host.OrgID).Msg("cannot get profile from database")
-				return
-			}
+		profile, err := db.GetOrInsertCurrentProfile(event.Host.OrgID, db.NewProfile(event.Host.OrgID, event.Host.Account, defaultState))
+		if err != nil {
+			logger.Error().Err(err).Str("org_id", event.Host.OrgID).Msg("cannot get profile from database")
```

Comment:
Will it be possible to include `request_id` in this and following log statements?  Thoughts?

Reply:
yes, we can.